src/backend/mailbox_manager/management/commands/setup_dimail_db.py

```diff
@@ -157,3 +172,39 @@ def create_allows(self, user, domain, auth=(regie["username"], regie["password"]
                      ........ failed: {response.json()['detail']}"
                 )
             )
+
+    def _populate_dimail_from_people(self):
+        self.stdout.write("Creating accounts from people database", ending="\n")
+
+        user_to_create = set()
+        domain_to_create = set()
+        access_to_create = set()
+        for mail_access in MailDomainAccess.objects.select_related(
+            "domain", "user"
+        ).all():
+            user_to_create.add(mail_access.user)
+            domain_to_create.add(mail_access.domain)
+            access_to_create.add(mail_access)
+
+        # create missing domains
+        for domain in domain_to_create:
+            # enforce domain status
+            if domain.status != MailDomainStatusChoices.ENABLED:
+                self.stdout.write(
+                    f"  - {domain.name} status {domain.status} -> ENABLED"
+                )
+                domain.status = MailDomainStatusChoices.ENABLED
+                domain.save()
+            self.create_domain(domain.name)
+
+        # create missing users
+        for user in user_to_create:
+            self.create_user(
+                auth=(admin["username"], admin["password"]),
+                name=user.sub,
+                perms=["new_domain", "create_users", "manage_users"],  # give all perms
```

Comment:
in dimail, we give no perms to basic users. Of the dimail users we're handling, only la_regie has these perms. 

I suggest reducing perms to [] to be iso with staging / preprod and prod environments.